rust-symcrypt/src/hash.rs

```diff
@@ -251,9 +282,9 @@ impl Drop for Md5State {
         unsafe {
             // SAFETY: FFI calls
             symcrypt_sys::SymCryptWipe(
-                ptr::addr_of_mut!(self.0) as *mut c_void,
-                mem::size_of_val(&mut self.0) as symcrypt_sys::SIZE_T,
-            )
+                self.get_inner_mut() as *mut c_void,
+                mem::size_of_val(&*self.get_inner()) as symcrypt_sys::SIZE_T,
```

Comment:
Did you debug breakpoint here to confirm?

Reply:
Confirmed with debugger, also confirmed ref counting for Hmac Expanded Key